packages/webapp/app/providers.tsx

```diff
@@ -6,19 +6,37 @@ import { QueryClientProvider } from '@tanstack/react-query';
 import { getQueryClient } from '@dailydotdev/shared/src/graphql/queryClient';
 import { AppSubscriptionContextProvider } from '@dailydotdev/shared/src/contexts/SubscriptionContext';
 import { ProgressiveEnhancementContextProvider } from '@dailydotdev/shared/src/contexts/ProgressiveEnhancementContext';
+import { BootApp } from '@dailydotdev/shared/src/lib/boot';
+import { BootDataProvider } from '@dailydotdev/shared/src/contexts/BootProvider';
+import useDeviceId from '@dailydotdev/shared/src/hooks/log/useDeviceId';
+import { PixelsProvider } from '../context/PixelsContext';
+import useWebappVersion from '../hooks/useWebappVersion';
+import { getPage, getRedirectUri } from '../pages/_app';
 
 export default function Providers({
   children,
 }: PropsWithChildren): ReactElement {
   const [queryClient] = useState(() => getQueryClient());
+  const version = useWebappVersion();
+  const deviceId = useDeviceId();
 
   return (
-    <QueryClientProvider client={queryClient}>
-      <ProgressiveEnhancementContextProvider>
-        <AppSubscriptionContextProvider>
-          {children}
-        </AppSubscriptionContextProvider>
-      </ProgressiveEnhancementContextProvider>
-    </QueryClientProvider>
```

Comment:
We need the `LogContext`

Reply:
Where? because it's already inside one of these no?
Afraid this might give some edge-case issue where context wrong order now.

Reply:
All those are inside the `BootDataProvider` so i removed the dupes